Paco-Server/src/com/google/sampling/experiential/server/CloudSQLDaoImpl.java

```diff
@@ -0,0 +1,326 @@
+package com.google.sampling.experiential.server;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Logger;
+
+import org.joda.time.DateTime;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.google.sampling.experiential.model.Event;
+import com.google.sampling.experiential.shared.EventDAO;
+import com.pacoapp.paco.shared.model2.EventBaseColumns;
+import com.pacoapp.paco.shared.model2.OutputBaseColumns;
+
+public class CloudSQLDaoImpl implements CloudSQLDao{
+  public static final Logger log = Logger.getLogger(CloudSQLDaoImpl.class.getName());
+  private static  Map<String, Integer> eventsOutputColumns = null;
+
+  private static void loadColumnTableAssociationMap(){
+    if (eventsOutputColumns ==null){
+      eventsOutputColumns = new HashMap<String,Integer>();
+      eventsOutputColumns.put("EXPERIMENT_ID", 1);
+      eventsOutputColumns.put("EXPERIMENT_NAME", 2);
+      eventsOutputColumns.put("EXPERIMENT_VERSION", 3);
+      eventsOutputColumns.put("SCHEDULE_TIME", 4);
+      eventsOutputColumns.put("RESPONSE_TIME", 5);
+      eventsOutputColumns.put("GROUP_NAME", 6);
+      eventsOutputColumns.put("ACTION_TRIGGER_ID",7);
+      eventsOutputColumns.put("ACTION_TRIGGER_SPEC_ID", 8);
+      eventsOutputColumns.put("ACTION_ID", 9);
+      eventsOutputColumns.put("WHO", 10);
+      eventsOutputColumns.put("PACO_VERSION", 11);
+      eventsOutputColumns.put("EVENTS._ID", 12);
+      eventsOutputColumns.put("_ID", 13);
+      eventsOutputColumns.put("TEXT", 14);
+      eventsOutputColumns.put("ANSWER", 15);
+    }
+  }
+ 
+  @Override
+  public boolean insertEvent(Event event) throws SQLException{
+    Connection conn = null;
+    PreparedStatement statementCreateEvent = null;
+    PreparedStatement statementCreateEventOutput = null;
+    
+    if (event == null){
+      log.warning("nothing to insert");
+      return false;
+    }
+    
+    try{
+      log.info("inserting event->"+ event.getId());
+      conn = CloudSQLConnectionManager.getInstance().getConnection();
+      String insertEventSql = "INSERT INTO events ("
```

Comment:
Would it be useful to use the Jsqlparser sql builder api to build the sql statements like this? Not sure if it would create more overhead?